Dragonflight/WarriorArms.lua

```diff
@@ -530,10 +566,12 @@ spec:RegisterHook( "reset_precast", function ()
     if prev_gcd[1].colossus_smash and time - action.colossus_smash.lastCast < 1 and last_cs_target == target.unit and debuff.colossus_smash.down then
         -- Apply Colossus Smash early because its application is delayed for some reason.
         applyDebuff( "target", "colossus_smash" )
+        if talent.test_of_might.enabled then state:QueueAuraExpiration( "test_of_might", TriggerTestOfMight, debuff.colossus_smash.expires ) end
     elseif prev_gcd[1].warbreaker and time - action.warbreaker.lastCast < 1 and last_cs_target == target.unit and debuff.colossus_smash.down then
         applyDebuff( "target", "colossus_smash" )
+        if talent.test_of_might.enabled then state:QueueAuraExpiration( "test_of_might", TriggerTestOfMight, debuff.colossus_smash.expires ) end
```

Comment:
No need to nest these checks for each of these "virtual" ToM expirations.

`if debuff.colossus_smash.up and talent.test_of_might.enabled then state:QueueAuraExpiration...`

Reply:
denested in https://github.com/Hekili/hekili/pull/1670/commits/55ab435a9c68e2a831358e7ed711a75f1affae57